### PR TITLE
fix: adapt uds server auxiliary

### PR DESCRIPTION
**Pull request by yannpoupon**

### CODE REVIEW

File: src/pykiso/lib/auxiliaries/udsaux/uds_server_auxiliary.py
Comment (sebastianpfischer) [inline src/pykiso/lib/auxiliaries/udsaux/uds_server_auxiliary.py:133]: parameter description missing

**Review by Pog3k — APPROVED** [review]

**Review by sebastianpfischer — APPROVED** [review]

**Comment by sebastianpfischer** [comment]
Somehow our ci/cd pipeline keeps failing. Please double-check the state of the coverage before clicking the merge button.

**Comment by yannpoupon** [comment]
> Somehow our ci/cd pipeline keeps failing. Please double-check the state of the coverage before clicking the merge button.

The pipeline is failing on master branch also with a test failing: https://ci.eclipse.org/kiso-testing/job/kiso-testing/job/master/250/

**Comment by sebastianpfischer** [comment]
> > Somehow our ci/cd pipeline keeps failing. Please double-check the state of the coverage before clicking the merge button.
> 
> The pipeline is failing on master branch also with a test failing: https://ci.eclipse.org/kiso-testing/job/kiso-testing/job/master/250/

@yannpoupon , there is a parallel PR to review to fix that.

**Pull request merged**
